dts/bindings/dma/nxp,mcux-edma.yaml

```diff
@@ -33,22 +38,11 @@ properties:
     required: true
     description: Number of items to expect in a DMAMUX specifier
 
-# Parameter syntax of NXP follows the dmamux client dts syntax
-# in the Linux kernel declared in
-# https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/dma/fsl-edma.txt
-#
 # - #dma-cells : Must be <2>.
-#  The 1st cell specifies the DMAMUX(0 for DMAMUX0 and 1 for DMAMUX1).
-#    Specific request source can only be multiplexed by specific channels
-#    group called DMAMUX.
+#  The 1st cell specifies the DMA channel (not DMAMUX channel) which will be used
```

Comment:
this is not going to work in our legacy platfrom there are two DMA mux with the same channell numbers. I would suggest that you can define a new v3 dt yaml file, and includes both of them.

Reply:
If i understand correctly, the current bindings for _dma-cells_ was copied from Linux's implementation. But the implementation in Zephyr is difference as mentioned in the commit msg.

Could you pls give me the name of SoC that will not work with this update? look like current boards in upstream are not be impacted

Reply:
> If i understand correctly, the current bindings for _dma-cells_ was copied from Linux's implementation. But the implementation in Zephyr is difference as mentioned in the commit msg.

it is referenced from linxu dma dts, but we need change for the real case. the impacted platforms are rt11xx seriese, which has 2 edma mux.

Reply:
@hakehuang is this still a concern for you?